.travis.yml

```diff
@@ -50,8 +50,6 @@ matrix:
 # 7.2
     - php: 7.2
       env: TEST_CONFIG="phpunit.xml"
-    - php: 7.2
-      env: ELASTICSEARCH_VERSION="1.4.2" TEST_CONFIG="phpunit-integration-legacy-elasticsearch.xml"
```

Comment:
Note to self, we can backport this part to 6.7 and up given we never officially supported.
Hence align and simplify travis setup to not spend CI time on long time deprecated impl.

Reply:
> Note; IMHO we can backport this part to 6.7 and up given we never officially supported it.
> Hence align and simplify travis setup to not spend CI time on long time deprecated impl.

#2627 :)